package.json

```diff
@@ -41,41 +41,41 @@
     "ember-auto-import": "^1.11.3",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1",
+    "ember-cli-showdown": "^6.0.0",
```

Comment:
Moved to `dependencies` as the `MarkdownToHtml` component is used for the `FreestyleShowdownContent` component.

Reply:
Hmm, my intent is to make this a dev dependency and include information in the docs on how to do this in userland. I think we should remove FreestyleShowdownContent and update the markdown section of `tests/dummy/app/templates/documentation.hbs` to mention this.

Reply:
Oh okay, shall I move it back to devDeps for this PR and create a separate ticket for deprecating/removing `FreestyleShowdownContent`?

Reply:
Dropped the commit that moved `ember-cli-showdown` and created https://github.com/chrislopresto/ember-freestyle/issues/608 as a follow-up issue.